src/arc.ts

```diff
@@ -110,18 +116,41 @@ export function providerHasConfigUi(provider?: any): any | undefined {
   return provider && provider.isTorus;
 }
 
+async function getAllDaos(arc: Arc): Promise<any> {
```

Comment:
Please, specify the retunring type here like:
```ts
async function getAllDaos(arc: Arc): Promise<Array<{id: string; name: string;}>> {
  // body
}
```

Reply:
I've added an interface and it looks like this `async function getAllDaos(arc: Arc): Promise<Array<IDAO>>`